packages/teams-js/src/internal/communication.ts

```diff
@@ -368,7 +369,9 @@ export function sendNestedAuthRequestToTopWindow(message: string): NestedAppAuth
   const targetWindow = Communication.topWindow;
   const request = createNestedAppAuthRequest(message);
 
-  logger('Message %i information: %o', request.uuid, { actionName: request.func });
+  logger('Message %s (legacy id: %i) information: %o', request.uuid.toString(), request.id, {
```

Comment:
How would you feel about some sort of helper function that takes in request and spits out `%s (legacy id: %i)`? That would be nice both from a reduce typing point of view but also from a "what if we change what we think of as the request's id in the future" point of view

Reply:
Done